src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -91,7 +92,8 @@ private String getQueueCheckLastexecKey() {
 
     // Consumers periodically refresh their subscription while they are
     // consuming.
-    private int refreshPeriod = 10;
+    private int refreshPeriod;
```

Comment:
Note: the variable `refreshPeriod` is set unconditionally in `start()` method --> no need for an initial value '10' here.